cmd/version.go

```diff
@@ -10,23 +10,28 @@ import (
 	"go.k6.io/k6/lib/consts"
 )
 
+func versionString() string {
+	v := consts.FullVersion()
+
+	if exts := ext.GetAll(); len(exts) > 0 {
+		extsDesc := make([]string, 0, len(exts))
+		for _, e := range exts {
+			extsDesc = append(extsDesc, fmt.Sprintf("  %s", e.String()))
+		}
+		v += fmt.Sprintf("\nExtensions:\n%s\n",
+			strings.Join(extsDesc, "\n"))
+	}
+	return v
+}
+
 func getCmdVersion(gs *state.GlobalState) *cobra.Command {
 	// versionCmd represents the version command.
 	return &cobra.Command{
 		Use:   "version",
 		Short: "Show application version",
 		Long:  `Show the application version and exit.`,
 		Run: func(_ *cobra.Command, _ []string) {
-			printToStdout(gs, fmt.Sprintf("k6 v%s\n", consts.FullVersion()))
-
-			if exts := ext.GetAll(); len(exts) > 0 {
-				extsDesc := make([]string, 0, len(exts))
-				for _, e := range exts {
-					extsDesc = append(extsDesc, fmt.Sprintf("  %s", e.String()))
-				}
-				printToStdout(gs, fmt.Sprintf("Extensions:\n%s\n",
-					strings.Join(extsDesc, "\n")))
-			}
+			printToStdout(gs, fmt.Sprintf("k6 v%s\n", versionString()))
```

Comment:
```suggestion
	root := cmd.Root()
    root.SetArgs([]string{"--version"})
	root.Execute()
```
This should do the trick for running a `k6 version` as `k6 --version`